docs/production-deployment/cloud/multi-region.mdx

```diff
@@ -364,8 +378,9 @@ The following sections show you how to perform this action, moving your active N
 You'll also read about what to expect after a failover.
 
 :::warning Check Your Replication Lag
-- Always check the [metric replication lag](https://temporal-documentation-clljyhxqv.preview.thundergun.io/production-deployment/cloud/metrics/reference#temporal_cloud_v0_replication_lag_bucket) before initiating a failover.
-  A forced failover when there is a large replication lag has a higher likelihood of rolling back Workflow progress.
+
+Always check the [metric replication lag](/production-deployment/cloud/metrics/reference#temporal_cloud_v0_replication_lag_bucket) before initiating a failover.
```

Comment:
Should this point to the metric or the new how-to about monitoring replication lag?

Reply:
I'm agnostic. I expect we'll move some of this content, as you noted above.